python/oneflow/test/modules/test_broadcast_like.py

```diff
@@ -72,6 +72,38 @@ def _test_broadcast_like_4dim(test_case, device):
     test_case.assertTrue(np.allclose(of_out.numpy(), np_out, 1e-05, 1e-05))
 
 
+def _test_broadcast_like_different_dim(test_case, device):
+    input = flow.tensor(
+        np.ones(shape=(3, 1), dtype=np.float32),
+        dtype=flow.float32,
+        device=flow.device(device),
+    )
+    like_tensor = flow.tensor(
+        np.ones(shape=(2, 3, 4), dtype=np.float32),
+        dtype=flow.float32,
+        device=flow.device(device),
+    )
+    of_out = flow.broadcast_like(input, like_tensor, broadcast_axes=(0, 2))
```

Comment:
```suggestion
    of_out = flow.broadcast_like(input, like_tensor)
```

这个 broadcast_axes=(0, 2) 不指定了吧，直接让他自己推导